Gopkg.toml

```diff
@@ -2,6 +2,10 @@
   name = "github.com/golang/glog"
   revision = "23def4e6c14b4da8ac2ed8007337bc5eb5007998"
 
+[[constraint]]
+  name = "github.com/prometheus/client_golang"
+  branch = "master"
```

Comment:
Can we use a tag instead of `master`? Maybe `version = 0.8.0`?

Reply:
Makes sense.